queries/kotlin/highlights.scm

```diff
@@ -0,0 +1,162 @@
+;;; Kotlin 
+;; Issues
+
+; mapOf with type definition AND multiple entries are not highligted correctly. Issue in parser. Eg.
+; a = mapOf<String, String>(AA.value to AA.value, AA.value to AA.value)
+
+
+(function_modifier) @type
+
+;; String
+
+(line_string_literal) @string
+(multi_line_string_literal) @string
+(line_string_literal (interpolated_identifier)) @string
```

Comment:
interpolated_identifier can be `@none` and `@variable`. See dart highlights.

Reply:
how is this different from line 12? But you can give highlights to `interpolated_identifier`

Reply:
clean up this part a bit in the latest commit